public/app/AppWrapper.tsx

```diff
@@ -107,18 +107,20 @@ export class AppWrapper extends React.Component<AppWrapperProps, AppWrapperState
                 >
                   <ModalsProvider>
                     <GlobalStyles />
-                    {config.featureToggles.commandPalette && <CommandPalette />}
+                    {!config.isPublicDashboardView && config.featureToggles.commandPalette && <CommandPalette />}
```

Comment:
```suggestion
                    {this.commandPaletteEnabled() && <CommandPalette />}
```

just to keep the view simpler and easier to read